File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.js
function solution(arr1, arr2) {
    var answer = [];
    
    const col = arr1.length;
    const row = arr2[0].length;
    const len = arr1[0].length;
    
    for(let i= 0; i<col; i++) {
        let arr = [];
        
        for(let j=0; j<row; j++) {
            let num =0;
            for(let k=0; k<len; k++) {
                num += arr1[i][k] * arr2[k][j];
            }
            arr.push(num);
        }
        answer.push(arr);
    }
    
    return answer;
}

File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.cpp
#include <string>
#include <vector>

using namespace std;

vector<vector<int>> solution(vector<vector<int>> arr1, vector<vector<int>> arr2) {
    vector<vector<int>> answer;
    
    int col = arr1.size();
    int len = arr1[0].size();
    int row = arr2[0].size();
    for(int i=0; i<col; i++)
    {
        vector<int> v;
        for(int j=0; j<row; j++) 
        {
            int num = 0;
            for(int k=0; k<len; k++)
            {
                num += arr1[i][k] * arr2[k][j];
            }
            v.push_back(num);
        }
        
        answer.push_back(v);
    }
    
    return answer;
}